.github/workflows/tests.yml

```diff
@@ -65,3 +65,11 @@ jobs:
       - name: yarn test:consumer
         if: matrix.node-version == '14.x'
         run: yarn test:consumer
+
+      - name: install bf-cli
+        if: matrix.node-version == '14.x'
+        run: npm install -g --ignore-scripts bf-cli@latest
```

Comment:
Ideally we would add this as a dev dependency in the root `package.json` file. That ensures that it's installed via `yarn` and cached which speeds up subsequent CI runs. Is that possible?

Reply:
It is. It felt a little odd to install this whole package for all devs. Would it make more sense to `yarn add bf-cli -W` as a GH Actions step?